Aptoide/appcoins-wallet-android
# [APPC-1126] Transactions screen incorrect balance

**What does this PR do?**

   Fixed some rounding problems (in bonus, balance and main screen, issue related in the ticket) and removed some unused classes

**Where should the reviewer start?**

-LocalCurrencyConversionService.java
-BalanceDetailMapper

**How should this be manually tested?**

  Check the use case in the ticket and check the top up bonus and see if it matches with what should be

**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/APPC-1126

**Code Review Checklist**
